plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/ext/panel/SubsectionPanel.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*global Ext, NX*/
+
+/**
+ * A panel that groups content by subsection.
+ *
+ * @since 3.0
+ */
+Ext.define('NX.ext.panel.SubsectionPanel', {
+  extend: 'Ext.panel.Panel',
+  alias: 'widget.nx-subsection-panel',
+
+  layout: {
+    type: 'vbox',
+    align: 'stretch'
+  },
+
+  padding: '12px',
+
+  autoScroll: true,
+
+  /**
+   * @override
+   */
+  listeners: {
+    add: function (me, component) {
+      var thisTitle = component.title || '',
+        thisWeight = component.weight || 1000,
+        position = 0;
+
+      me.suspendEvents();
+      me.remove(component, false);
+
+      me.items.each(function (item) {
+        var thatTitle = item.title || '',
+          thatWeight = item.weight || 1000;
+
+        if (thisWeight < thatWeight
+          || (thisWeight === thatWeight && thisTitle < thatTitle)) {
```

Comment:
I think it’s so tabs with the same weight are sorted alphabetically by default.

@adreghiciu ?